lib/jablotron.js

```diff
@@ -101,15 +103,25 @@ Jablotron.prototype = {
             if (sessionId && sessionId != null) {
                 self.client.doAuthenticatedRequest('/dataUpdate.json', payload, sessionId, function (response) {
                     let responseData = self.parseResponseData(response);
-                    if (responseData != null) {
-                        let segments = responseData[0]['data']['segments'];
+                    if (responseData["data"] != null) {
+                        let alarm_service_event = [];
+                        if (responseData["service_events_type"] == "tamper" || responseData["service_events_type"] == "alarm") {
```

Comment:
Be carefull with these attributes - they could be null for thermometers. Even though thermometers use the same structure majority of attributes returned is null